lib/arrays.md

```diff
@@ -5,44 +5,91 @@ instructor_notes: Feel free to re-organize the headings (or add/remove headings)
 
 # What is an Array?
 
-Replace me with your response...
+An ordered list of objects. List objects can be integers, strings, floats, hashes, symbols, and other arrays.
+Each object in the array has an associated index with the first object's index being 0.
 
 # What are some examples of information that would be Arrays as opposed to some other data type?
 
-Replace me with your response...
+A list a cat's favorite things. All 12 months. The alphabet. Dinosaur types. Prime numbers. 
 
 # What is one way, using Ruby, to retrieve the 6th element in an Array? How about the 8th element? What happens if you try to retrieve the value of the _9999th_ element (or some element that doesn't exist in the array)?
+Same concept as in strings. the first element in an array is in the 0 position, second is in the 1 position and so on.
+
+    prime = [2, 3, 5, 7, 11, 13, 17, 19, 23, 29, 31, 37, 41, 43, 47]
+    prime[6]
+     will return 17
+    prime[8]
+     will return 23
+    prime[99]
+     will return 'nil' because it does not exist
 
-Replace me with your response...
 
 # The previous question asks about finding, for example, the 6th element in an Array. Is it possible to find the **-6th** (Notice the negative symbol!) element in an Array? What does that even mean?
 
-Replace me with your response...
+Same concept as in strings. Last element in array will be in the -1 position, second to last will be in the -2 position, and so on.
 
+    prime[-6]
+     will return 29
+    
 # How would you perform an operation on every element inside an Array?
 
-Replace me with your response...
+    prime.each do |x|
+     puts x+=1
+    end
+    
+This would add 1 to each element in the array
 
 # How do you add elements to an Array?
 
-Replace me with your response...
+You can insert something in a specific location:
+
+    prime.insert(-1, 53)
+     inserts 53 in the -1 position of the array
+    prime << 53
+     puts 53 at the last positon of the array
+    prime.push(53).push(59)
+     pushes the new elements to the end of the array. Able to chain multiple pushes together
 
 # Given the Array `["Laura", "Fiona", "Tori"]`, how would you replace `"Fiona"` with `"Florence"` so that you end up with `["Laura", "Florence", "Tori"]`?
+The .map method creates a new array based on the original array, but changes the elements based on your block.
 
-Replace me with your response...
+    arr = ["Laura", "Fiona", "Tori"]
+    arr.map! { |element|
+       if(element == "Fiona")
+           "Florence"
+       else
+           element
+       end
+    }
```

Comment:
Creative! I like the thought process, but I'd also like for you to find the more straightforward way to do this.

The task is much simpler than your answer might indicate: You're trying to change the value of an element in an array at a given position. You already know how to reference an element by its given position, so setting that equal to something will take no more than one line of code.

I'm being deliberately opaque here, so please feel free to follow up if you're still unclear after some research and thought.